pose_pipeline/env.py

```diff
@@ -22,27 +31,9 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 
 def set_environmental_variables():
-    # TODO: should create a cfg file or use a path relative to module for this instead
-    # of hardcoding for my local setup
-    os.environ["OPENPOSE_PATH"] = "/home/jcotton/projects/pose/openpose"
-    os.environ["OPENPOSE_PYTHON_PATH"] = "/home/jcotton/projects/pose/openpose/build/python"
-    os.environ["EXPOSE_PATH"] = "/home/jcotton/projects/pose/expose"
-    os.environ["CENTERHMR_PATH"] = "/home/jcotton/projects/pose/CenterHMR"
-    os.environ["GAST_PATH"] = "/home/jcotton/projects/pose/GAST-Net-3DPoseEstimation"
-    os.environ["POSEFORMER_PATH"] = "/home/jcotton/projects/pose/PoseFormer"
-    os.environ["VIBE_PATH"] = "/home/jcotton/projects/pose/VIBE"
-    os.environ["MEVA_PATH"] = "/home/jcotton/projects/pose/MEVA"
-    os.environ["PARE_PATH"] = "/home/jcotton/projects/pose/PARE"
-    os.environ["PIXIE_PATH"] = "/home/jcotton/projects/pose/PIXIE"
-    os.environ["HUMOR_PATH"] = "/home/jcotton/projects/pose/humor/humor"
-    os.environ["FAIRMOT_PATH"] = "/home/jcotton/projects/pose/FairMOT/src/lib"
-    os.environ["DCNv2_PATH"] = "/home/jcotton/projects/pose/DCNv2/DCN"
-    os.environ["TRANSTRACK_PATH"] = "/home/jcotton/projects/pose/TransTrack"
-    os.environ["PROHMR_PATH"] = "/home/jcotton/projects/pose/ProHMR"
-    os.environ["TRADES_PATH"] = "/home/jcotton/projects/pose/TraDeS/src/lib"
-    os.environ["RIE_PATH"] = "/home/jcotton/projects/pose/Pose3D-RIE"
-    os.environ["VIDEOPOSE3D_PATH"] = "/home/jcotton/projects/pose/VideoPose3D"
-    os.environ["POSEAUG_PATH"] = "/home/jcotton/projects/pose/PoseAug"
+    """For dependency listed below, checks that path exists and sets env variable."""
+    os.environ["POSE_PROJ_DIR"] = get_pose_project_dir()
+    load_dotenv()
```

Comment:
This is definitely an improvement on the existing hard-coded approach. For future distribution, do you think we will want to use the default path though? This assumes that when the user runs this command, they have the correct .env file in their PWD. Perhaps `set_environmental_variables` should have an optional path, and if that is absent default to whether the package is installed?

Reply:
Sure - I'll supply a default that aligns with your local setup (i.e. `/home/jcotton/...`) and allow a user to specify an alternative in a function argument if they don't want to use the datajoint config to do so